src/library/crate/cratestorage.cpp

```diff
@@ -410,6 +410,19 @@ QString CrateStorage::formatSubselectQueryForCrateTrackIds(
             crateId.toString());
 }
 
+//static
+QString CrateStorage::formatQueryForTrackIdsByCrateName(
+        const QString& crateName) {
+    return QString("SELECT %1 FROM %2 JOIN %3 ON %4 = %5 WHERE %6 LIKE %7").arg(
```

Comment:
You need to use SELECT DISTINCT to avoid duplicate TrackIds if the LIKE statement matches multiple crates.